lib/api/canto/index.ts

```diff
@@ -0,0 +1,67 @@
+import { fallbackLng } from "@/lib/i18n/settings";
+import { resizeCantoImage, ValidCantoSizes } from "./resize";
+
+const responsiveCantoSrc = (
```

Comment:
move over some Canto helpers from the investigations that we can start using for pages whose opengraph image is a Canto asset